examples/plot_mixed_gambles.py

```diff
@@ -0,0 +1,80 @@
+"""
+GLM fitting in second level fMRI
+================================
+
+Full step-by-step example of fitting a GLM to perform a second level analysis
+in experimental data and visualizing the results.
+
+More specifically:
+
+1. A sequence of fMRI volumes are loaded
+2. A design matrix describing all the effects related to the data is computed
+3. a mask of the useful brain volume is computed
+4. A GLM is applied to the dataset (effect/covariance,
+   then contrast estimation)
+
+"""
+print(__doc__)
+
+import os
+from os import mkdir, path
+
+import numpy as np
+import pandas as pd
+from nilearn import plotting
+import nibabel as nib
+
+from nilearn.datasets import fetch_mixed_gambles
+from nistats.second_level_model import SecondLevelModel
+
+# write directory
+write_dir = 'results'
+first_level_paths = os.path.join('results', 'first_level_imgs')
+if not path.exists(first_level_paths):
+    os.makedirs(first_level_paths)
+
+# get data ###################################################
+data = fetch_mixed_gambles(16)
+zmap_paths = []
+for zidx, zmap in enumerate(data.zmaps):
+    zmap_path = os.path.join(first_level_paths, 'map_%03d.nii' % zidx)
+    nib.save(zmap, zmap_path)
+    zmap_paths.append(zmap_path)
+behavioral_target = np.ravel(['gain_%d' % (i + 1) for i in data.gain]).tolist()
+subjects_id = np.ravel([['sub_%02d' % (i + 1)] * 48 for i in range(16)])
+subjects_id = subjects_id.tolist()
+mask_filename = data.mask_img
+
+
+# Second level model #########################################
+# create first_level_input
```

Comment:
this comment is a bit misleading. "create input for second-level analysis" sounds better.